### PR TITLE
Minor grammar changes for Romi docs

### PR DESCRIPTION
**Pull request by rshah713**
Small grammatical changes & typos that stood out when reading Romi docs.

### CODE REVIEW

File: source/docs/romi-robot/getting-to-know-romi.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ In all Romi documentation, references to driving forward use the above definitio
 .. image:: images/getting-to-know-romi/romi-forward.png
    :alt: Romi Forward Driving Direction
 
-Hardware, Sensors and GPIO
+Hardware, Sensors, and GPIO
 --------------------------
```
Comment (Daltz333) [inline source/docs/romi-robot/getting-to-know-romi.rst:18]:
```suggestion
---------------------------
```

File: source/docs/romi-robot/getting-to-know-romi.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ The Romi has the following built-in hardware/peripherals:
 
 .. note:: The Buzzer is currently not supported by WPILib.
 
-Motors, Wheels and Encoders
+Motors, Wheels, and Encoders
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (Daltz333) [inline source/docs/romi-robot/getting-to-know-romi.rst:32]:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Review by Daltz333 — COMMENTED** [review]
Regarding Oxford commas. I typically don't see this as an issue in the documentation. I'm fine merging this as is, but I wouldn't recommend looking for places to fix.

**Comment by rshah713** [comment]
Agreed, no need to look for further places to fix. Feel free to merge PR as is.

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**